### PR TITLE
update thci for BBR

### PR DESCRIPTION
**Pull request by tomaszkob89**

### CODE REVIEW

File: thci/OTNCS.py [hunk 1]
```diff
@@ -3602,9 +3607,6 @@ def _cliReadLine(self):
             return self.__lines.pop(0)
 
     def _cliWriteLine(self, line):
-        # NCS adaptation
```
Comment (edmont) [inline thci/OTNCS.py]: I think this was added for a reason. Might be breaking some Thread 1.1 certification test by removing those lines.

File: thci/OTNCS_BR.py [hunk 2]
```diff
@@ -38,7 +43,9 @@
 import serial
 from GRLLibs.UtilityModules.ModuleHelper import ModuleHelper
 from IThci import IThci
-from THCI.OpenThread import OpenThreadTHCI, watched, API
+from THCI.OTNCS import OpenThreadTHCI, watched, API
+
+# from THCI.OpenThread import OpenThreadTHCI, watched, API
```
Comment (edmont) [inline thci/OTNCS_BR.py]: This line can be removed.

**Review by edmont — COMMENTED** [review]
A couple of comments.

File: thci/OTNCS.py (hunk 1 above)
Comment (tomaszkob89) [reply thci/OTNCS.py]: restored

File: thci/OTNCS_BR.py (hunk 2 above)
Comment (tomaszkob89) [reply thci/OTNCS_BR.py]: line removed

File: thci/OTNCS.py (hunk 1 above)
Comment (edmont) [reply thci/OTNCS.py]: Doesn't this interfere with the BBR operation?

Comment (tomaszkob89) [reply thci/OTNCS.py]: OTNCS class is not used by OTNCS_BR class. The second has it's own implementation.

**Pull request merged**